api/scpca_portal/models/dataset.py

```
@@ -671,9 +671,28 @@ def original_file_paths(self) -> Set[Path]:
         return {Path(of.s3_key) for of in self.original_files}
 
     @property
-    def computed_file_name(self) -> Path:
-        return Path(f"{self.pk}.zip")
+    def computed_file_name(self) -> str:
+        if self.is_ccdl:
+            file_name = ccdl_datasets.COMPUTED_FILE_NAMES[self.ccdl_name]
+            return f"{file_name}.zip"
+
+        return f"{self.pk}.zip"
 
     @property
     def computed_file_local_path(self) -> Path:
         return settings.OUTPUT_DATA_PATH / self.computed_file_name
+
+    @property
+    def download_url(self) -> str:
+        """A temporary URL from which the file can be downloaded."""
+        if not self.computed_file or not self.computed_file.s3_absolute_path:
+            raise ValueError("Invalid download url request: No Computed File")
```

Do we want to raise an exception when processing download_url if there's no computed file, or should we just return None